po/Makefile

```diff
@@ -0,0 +1,33 @@
+APPNAME=kano-updater
+
+MSGLANGS=$(notdir $(wildcard *po))
+MSGOBJS=$(addprefix ../locale/,$(MSGLANGS:.po=/LC_MESSAGES/$(APPNAME).mo))
+
+.PHONY: clean messages
+
+build: $(MSGOBJS)
+
+update: $(MSGLANGS)
+
+clean:
+	rm -f messages.pot
+	rm -rf ../locale
+
+define run-xgettext
+xgettext -f PYPOTFILES -L Python --force-po -o messages.pot
+xgettext -f CPOTFILES -L C -k_ -kN_ -j -o messages.pot
+endef
+
+messages:
+	$(run-xgettext)
+
+messages.pot:
+	$(run-xgettext)
+
+%.po: messages.pot
+	msgmerge -N -U $*.po messages.pot
```

Comment:
After a chat with @skarbat, it seems that we encountered this same `msgmerge` problem last year when we were trying to build `openbox`. It is apparently an unsupported QEmu emulation call and the solution before was to just comment out the `msgmerge` line.

@skarbat suggested that this problem may be fixed since then, it was in the `qemu-user-static` package. There is potential that updating this *may* fix the problem but it might disrupt the build environment for our OS build. @pazdera any thoughts?